waves-ext/src/main/scala/com/wavesplatform/transaction/smart/script/ScriptRunnerFixed.scala

```diff
@@ -0,0 +1,136 @@
+package com.wavesplatform.transaction.smart.script
+
+import cats.Id
+import cats.implicits._
+import com.wavesplatform.account.AddressScheme
+import com.wavesplatform.common.state.ByteStr
+import com.wavesplatform.features.BlockchainFeatures
+import com.wavesplatform.lang._
+import com.wavesplatform.lang.contract.DApp
+import com.wavesplatform.lang.directives.DirectiveSet
+import com.wavesplatform.lang.directives.values.{Account, Asset, Expression, V3}
+import com.wavesplatform.lang.script.v1.ExprScript
+import com.wavesplatform.lang.script.{ContractScript, Script}
+import com.wavesplatform.lang.v1.compiler.Terms.{EVALUATED, TRUE}
+import com.wavesplatform.lang.v1.evaluator._
+import com.wavesplatform.lang.v1.evaluator.ctx.EvaluationContext
+import com.wavesplatform.lang.v1.evaluator.ctx.impl.waves.Bindings
+import com.wavesplatform.lang.v1.traits.Environment
+import com.wavesplatform.state._
+import com.wavesplatform.transaction.smart.{DApp => DAppTarget, _}
+import com.wavesplatform.transaction.{Authorized, Proven}
+import monix.eval.Coeval
+
+object ScriptRunnerFixed {
+  type TxOrd = BlockchainContext.In
+
+  def apply(
+    in: TxOrd,
+    blockchain: Blockchain,
+    script: Script,
+    isAssetScript: Boolean,
+    scriptContainerAddress: Environment.Tthis,
+    complexityLimit: Int = Int.MaxValue,
+    default: EVALUATED = TRUE
+  ): (Log[Id], Either[ExecutionError, EVALUATED]) =
+    apply(
```

Comment:
That method is for using in the Node.
We don't use it in the Matcher because we pass denied block-chain as the parameter.